apps/ledger-live-mobile/src/newArch/features/WalletSync/components/Synchronize/QrCode.tsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import { Flex, Text, NumberedList, ScrollContainer } from "@ledgerhq/native-ui";
+import styled, { useTheme } from "styled-components/native";
+import QRCode from "react-native-qrcode-svg";
+import getWindowDimensions from "~/logic/getWindowDimensions";
+import { Trans, useTranslation } from "react-i18next";
+
+const Italic = styled(Text)`
+  font-style: italic;
+`;
+// Won't work since we don't have inter italic font
+
+type Props = {
+  qrCodeValue: string;
+};
+
+const QrCode = ({ qrCodeValue }: Props) => {
+  const { colors } = useTheme();
+  const { width } = getWindowDimensions();
+  const { t } = useTranslation();
+
+  const QRSize = Math.round(width - 48);
+  const maxQRCodeSize = 280 - 15.36 * 2;
+  const QRCodeSize = Math.min(QRSize - 15.36, maxQRCodeSize);
```

Comment:
we may want to move some constants in a variable, especially 15.36 value, to explain what it is by a variable name